roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -107,8 +108,9 @@
       #!/bin/sh
       make_resolv_conf() { : ; }
 
-- name: Configure swap
-  ansible.builtin.import_tasks: swap.yml
+- name: Include swap tasks
+  ansible.builtin.include_tasks: swap.yml
+  when: not ansible_local.bootc
```

Comment:
@SeanMooney PTAL. Why can't we configure swap with bootc?

Reply:
swap is required for all comptue nodes so we will require swap even when using bootc

there are som either issues with this patch too.

for example it has regressed the ownership fo /var/lib/openstack which should always be owned by the ansibel user, never root

we have fixed that in the past on other roles like neutron.
https://github.com/openstack-k8s-operators/edpm-ansible/commit/e0a6307e1e9cfcb416d8928f51279b6ada27d9d8